openforcefield/utils/toolkits.py

```diff
@@ -160,6 +160,10 @@ class ChargeCalculationError(MessageException):
     pass
 
 
+class InvalidIUPACNameError(MessageException):  # TODO: Should this live in molecule.py?
```

Comment:
```suggestion
class InvalidIUPACNameError(MessageException):
```

I think this is the right place for it.